util/clustergen/schema/snitch_cluster.schema.json

```diff
@@ -203,6 +203,21 @@
                     "description": "Whether to expose memory cut configuration inputs for implementation",
                     "default": false
                 },
+                "wide_tcdm_port_expose": {
+                    "type": "boolean",
+                    "description": "Whether to expose a wide memory port",
```

Comment:
We can add more details, e.g.:

"Whether to expose a wide port into the TCDM at the cluster interface. Used to provide external masters, such as accelerators, with wide access to the TCDM."